webapp/stores/emoji_store.jsx

```diff
@@ -148,8 +149,7 @@ class EmojiStore extends EventEmitter {
 
     getEmojiImageUrl(emoji) {
         if (emoji.id) {
-            // must match Client.getCustomEmojiImageUrl
-            return `/api/v3/emoji/${emoji.id}`;
+            return Client.getCustomEmojiImageUrl(emoji.id);
```

Comment:
If we could get the existing tests working with the client for just this one case, that'd be good for now. I actually just found that we're looking into using Selenium for testing starting in the near future, but we haven't gotten that in place yet. In the meantime, we aren't able to merge this until the unit tests are passing.